dev/devicelab/lib/tasks/plugin_tests.dart

```diff
@@ -32,27 +32,35 @@ class PluginTest {
     this.pluginCreateEnvironment,
     this.appCreateEnvironment,
     this.dartOnlyPlugin = false,
+    this.template = 'plugin',
   });
 
   final String buildTarget;
   final List<String> options;
   final Map<String, String>? pluginCreateEnvironment;
   final Map<String, String>? appCreateEnvironment;
   final bool dartOnlyPlugin;
+  final String template;
 
   Future<TaskResult> call() async {
     final Directory tempDir =
         Directory.systemTemp.createTempSync('flutter_devicelab_plugin_test.');
+    // FFI plugins do not have support for `flutter test`.
+    // `flutter test` does not do a native build.
```

Comment:
unrelated to this test, what is the user experience if I run `flutter test` on an app with an FFI plugin? Does it just fail to lookup the symbol?

Reply:
The FFI plugin template itself doesn't include any unit tests:

> `Test directory "test" not found.`

When adding a test, opening the dynamic library fails:

> `Shell: Invalid argument(s): Failed to load dynamic library '<...>':`

I created a tracking issue:

* https://github.com/flutter/flutter/issues/97056